images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -101,7 +101,11 @@ $toolsList = @(
     (Get-NewmanVersion),
     (Get-NvmVersion),
     (Get-PackerVersion),
+    ("Pollinate"),
```

Comment:
Why we don't print version of Pollinate and Sphinx?

Reply:
They don't have this option

Reply:
added version for Sphinx and removed Pollinate from docs.